### PR TITLE
Add track2 config to containerinstance

### PR DESCRIPTION
**Pull request by 00Kai0**

### CODE REVIEW

**Review by PhoenixHe-NV — APPROVED** [review]

**Comment by NullMDR** [comment]
@00Kai0 Any update on this PR?

**Comment by 00Kai0** [comment]
@NullMDR ,not now. We will wait until Oct 24th to merge track2 version.

**Pull request closed without merging**
